src/server-entry.js

```diff
@@ -19,6 +19,43 @@ const fetch = require('make-fetch-happen');
 
 const isDev = process.env.NODE_ENV !== 'production';
 
+let renderedConfig = '';
+let renderedNoscript = '';
+let renderedExternals = '';
+
+// This adds non-vue-rendered html strings to the request context.
+// These strings are added to the final html response using server/index.template.html
+function addRenderedHtml(context, config) {
+	// render config if it hasn't been rendered yet
+	if (!renderedConfig) {
+		renderedConfig = renderGlobals({ __KV_CONFIG__: config });
+	}
+	// render noscript tag if it hasn't been rendered yet
+	if (!renderedNoscript) {
+		renderedNoscript = noscriptTemplate(config);
+	}
+	// render externals if they haven't been rendered yet
+	if (!renderedExternals) {
+		// add OneTrust loader
+		if (config.oneTrust && config.oneTrust.enable) {
+			const key = `${config.oneTrust.key}${config.oneTrust.domainSuffix}`;
+			const src = `https://cdn.cookielaw.org/consent/${key}/otSDKStub.js`;
+			// eslint-disable-next-line max-len
+			renderedExternals += `<script type="text/javascript" data-domain-script="${key}" src="${src}"></script>`;
```

Comment:
Moving this here allows the browser to start loading the javascript for oneTrust without having to wait on executing the primary ui head script and processing the inserted script node.